modules/mocked_authentication/lib/mocked_credential/service.rb

```diff
@@ -0,0 +1,59 @@
+
+    module MockedAuthentication
+      module MockedCredential
+      class Service 
+        
+        def token(code)
+          code
+        end
+  
+        def user_info(token)
+          mocked_authorization_credential_information =  MockedAuthRedis.find(token)
```

Comment:
`MockCredentialInfo` was added so you can change this now
```suggestion
          mocked_authorization_credential_information =  MockCredentialInfo.find(token)
```